sign in/sing up.html

```diff
@@ -0,0 +1,48 @@
+<html>
+<head>
+    
+    
+    <title>
+    </title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    
+</head>
+<body>
+    <div class="login_box">
+        <img src="icons8-user-64.png">
+        <br>
+        <h1> Sign In</h1>
+        <form action="#">
+            <div class="input_box">
+           
+            <p></p>
```

Comment:
### \<p\> tag should not be empty
If it is empty, so it is useless. Please delete or fill with meaningful data.
same problem at line 26